zh-cn/review/reviewer/navigate.md

```diff
@@ -0,0 +1,34 @@
+# 查看 CL 的步骤
+
+## 总结
+
+现在您已经知道了 [Code Review 要点](looking-for.md)，那么管理分布在多个文件中的评论的最有效方法是什么？
+
+1. 变更是否有意义？它有很好的描述吗？
+1. 首先看一下变更中最重要的部分。整体设计得好吗？
+1. 以适当的顺序查看 CL 的其余部分。
+
+## 第一步：全面了解变更 {#step_one}
+
+查看 CL 描述和 CL 的一般用途。这种变更是否有意义？如果不应该变更，请立即回复，说明为什么不应该进行变更。当您拒绝这样的变更时，向开发人员建议应该做什么也是一个好主意。
+
+例如，您可能会说“很感谢您的努力，谢谢！但实际上，我们正朝着删除您在这里修改的 FooWidget 系统的方向演进，所以我们不想对它进行任何新的修改。不过，您来重构下新的 BarWidget 类怎么样？“
+
+请注意，审查者不仅拒绝了当前的 CL 并提供了替代建议，而且他们礼貌地做了。这种礼貌很重要，因为我们希望表明，即使不同意，我们也会相互尊重。
+
+如果您获得了多个您不想变更的 CL，您应该考虑重整开发团队的开发过程或外部贡献者的发布过程，以便在编写CL之前有更多的沟通。最好在他们完成大量工作之前告诉别人“不”，这些工作现在必须被抛弃或彻底重写。
+
+## 第二步：检查 CL 的主要部分 {#step_two}
+
+查找作为此 CL “主要”部分的文件。通常，逻辑变更最大的文件就是 CL 的主要部分。先看看这些主要部分。这有助于为 CL 的所有较小部分提供上下文，通常可以加速代码审查。如果 CL 太大而无法确定哪些部分是主要部分，请向开发人员询问您应该首先查看的内容，或者要求他们[将 CL 拆分为多个 CL](../developer/small-cls.md)。
+
```

Comment:
缺漏一段翻譯
> If you see some major design problems with this part of the CL, you should send those comments immediately, even if you don’t have time to review the rest of the CL right now. In fact, reviewing the rest of the CL might be a waste of time, because if the design problems are significant enough, a lot of the other code under review is going to disappear and not matter anyway.

如果在该部分发现存在一些主要的设计问题时，即使没有时间立即查看 CL 的其馀部分，也应立即留下评论告知此问题。 因为事实上，因为该设计问题足够严重的话，继续审查其馀部分很可能只是浪费宝贵时间，因为其他正在审查的程式可能都将无关紧或消失。